src/Zoom.php

```diff
@@ -53,6 +53,16 @@ public function getLoginUrl($state = '')
 
         throw new ZoomException("Error: getLoginUrl()");
     }
+    
+    public function createUserMeeting($id, $topic = 'New Meeting', $type = '1', $password = 'MTGPASS', $agenda = 'Start Meeting')
```

Comment:
Not sure if we should be defaulting to these values.

Reply:
ok i'll remove default values.  think i'll add the other params too.

Reply:
Good idea, thanks.

Reply:
Perhaps passing an array of meeting params, or creating a `ZoomMeeting` class or something like that, may reduce the number of params you need to pass. Also, this would solve the typical order-of-parameters dilemma.

With the array solution, you could then (in the future, if needed) add default parameters with array_merge.